### DISCUSSION TITLE
是否可增加最佳实践的文档

### DISCUSSION THREAD

**Discussion by xulz** [httprunner/httprunner#1338] [1 - 想法（Ideas）]
应用最广泛的接口工具应该是Postman，假如我现在有个项目，有客户端和服务端两个collection，现有的转换工具会把一个collection转换成两个测试用例（client_postman_collection_test.json和server_postman_collection_test.json）。

我的理解，postman里一个request应该对应一个testcase。
同时公共部分的header应该是提取出来的，header应该放在debugtalk.py函数实现（因为认证头总是变化的，而且client、server是两种头信息）。
base_url等和测试环境相关的应该是放在.env的（方便和Jenkins等集成，支持多环境）。

初始数据(造数据)有什么建议吗，能不能一个folder（类似testsuite）共用一个setup/hook？

类似这种项目共有的一些推荐实践，建议后续可以增加相关文档说明。

P.S. hrp还在学习中，可能理解不够准确全面

**Comment by debugtalk** [comment]
@xulz 感谢反馈，后面我们写一篇实践文档吧。
